### DISCUSSION TITLE
Why does split view open the page both in the right side panel and the main panel?

### DISCUSSION THREAD

**Discussion by Mugen2049** [toeverything/AFFiNE#7950] [Q&A]
When I try to open a different page in split view, it duplicates it the main, center panel. I can't open two separate pages, one in each panel. Can anyone help? 

<img width="958" alt="split-view-problem" src="https://github.com/user-attachments/assets/ab59cdd6-c22a-40c8-b33f-4fb208675edc">

**Comment by EYHN** [comment]
In the splitview case, you can click `all docs` to access other docs.

**Reply by Mugen2049** [reply]
I got it now, thank you. Is there no way to directly open an existing tab into split view? Or open directly from "All Docs" without duplicating the page in both panels, then place the curser in the panel you want the other page to show up, then click "All Docs", and finally select the page?

I read on the AFFiNE blog there was supposed to be a drag and drop feature in one of their past updates. That is drag a page into the side panel to open. But this doesn't work.

Anyway, thank you for your help.
